### PR TITLE
Cc hotfix

### PR DESCRIPTION
**Pull request by charles-cowart**
Minor change to Qiita start script that prevents non-master Qiita instances from instantiating a Watcher process. This prevents multiple instances from attempting to call ProcessingJob.complete() on the same job.

### CODE REVIEW

File: qiita_db/software.py [hunk 1]
```diff
@@ -1303,14 +1302,16 @@ def client_secret(self):
...
-            self.start_script, url)
+        cmd = 'source ~/.bash_profile; %s; %s "%s" "register" "ignored"' % (
+            self.environment_script, self.start_script, url)
         # this print is intentional as it will be stored in the internal
         # Qiita logs
         print 'Registering: %s, via %s' % (self.name, cmd)
-        p = Process(target=qdb.processing_job._system_call, args=(cmd,))
-        p.start()
+        p_out, p_err, rv = qdb.processing_job._system_call(cmd)
+        print p_out, p_err, rv
```
Comment (antgonza) [inline qiita_db/software.py]: rm this line

File: qiita_db/software.py [hunk 2]
```diff
@@ -1303,14 +1302,16 @@ def client_secret(self):
...
+        if rv != 0:
+            raise ValueError('Error %d: %s' % (rv, p_out))
+        print p_out, p_err, rv
```
Comment (antgonza) [inline qiita_db/software.py]: rm this line

File: qiita_db/processing_job.py [hunk 3]
```diff
@@ -327,7 +327,7 @@ def _system_call(cmd):
     license.
     """
     proc = Popen(cmd, universal_newlines=True, shell=True, stdout=PIPE,
-                 stderr=PIPE)
+                 stderr=PIPE, executable='/bin/bash')
```
Comment (antgonza) [inline qiita_db/processing_job.py]: is this change still required?

File: qiita_db/software.py (hunk 2 above)
Comment (charles-cowart) [reply qiita_db/software.py]: you got it.

File: qiita_db/software.py (hunk 1 above)
Comment (charles-cowart) [reply qiita_db/software.py]: yup!

File: qiita_db/processing_job.py (hunk 3 above)
Comment (charles-cowart) [reply qiita_db/processing_job.py]: rm +1.
